crimsobot/cogs/utilities.py

```diff
@@ -95,6 +103,10 @@ async def polltally(self, ctx: commands.Context, poll_id: Optional[str] = None)
                     poll_found = True
                     descr = message.embeds[0].description
                     reactions = message.reactions
+
+                    if message.embeds[0].image is not discord.Embed.Empty:
```

Comment:
If memory serves, Embed.Empty is a false-y value. `if not message.embeds[0].image` should be valid.